wiki/BanchoBot/en.md

```diff
@@ -1,154 +1,168 @@
 
-*Note: The user argument is case-insensitive, as usernames are case-insensitive.*
+- [Help](#help)
+- [Roll](#roll)
+- [Stats](#user-statistics)
+- [Where](#where)
+- [FAQ](#faq)
+- [Report](#report)
 
-### HELP
+### Help
 
 ```
-!HELP
+!help
 ```
 
-Shows a list of available BanchoBot commands.
-
-#### Usage Example
-
-*Notice: The `user` parameter for the `REPORT` command is missing.*
-
-```
-pippi: !help
-BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
-BanchoBot: WHERE <user>
-BanchoBot: STATS <user>
-BanchoBot: FAQ <item>|list
-BanchoBot: REPORT <reason> - call for an admin
-BanchoBot: REQUEST [list] - shows a random recent mod request
-BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
-```
+`!help` shows a list of all available BanchoBot commands. An example of sending this command is shown below:
 
-### ROLL
+<!--note for editors: the code block below reflects the exact response from banchobot -->
 
 ```
-!ROLL <number>
+13:00 pippi: !help
+13:00 BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+13:00 BanchoBot: WHERE <user>
+13:00 BanchoBot: STATS <user>
+13:00 BanchoBot: FAQ <item>|list
+13:00 BanchoBot: REPORT <reason> - call for an admin
+13:00 BanchoBot: REQUEST [list] - shows a random recent mod request
+13:00 BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
 ```
 
-Draws a random number from 1 to the selected number (default is 100).
+### Roll
 
-#### Usage Example
+*Note: players may add an argument after `!roll` and before `<number>`, and the command will still be understood. Although it does not change the outcome.*
 
 ```
-pippi: !roll 1000
-BanchoBot: pippi rolls 109 point(s)
+!roll <number>
 ```
 
-### STATS
+`!roll` draws a random number from 1 to the selected number (default is 100). An example of sending this command is shown below:
 
 ```
-!STATS <user>
+13:00 pippi: !roll 1000
+13:00 BanchoBot: pippi rolls 109 point(s)
 ```
 
-Shows the entered user's stats. The resulting stats depend on the mode they are playing or last played.
+### User statistics
 
-*Note: BanchoBot will not tell you which game mode the stats came from.*
-
-*Notice: BanchoBot will respond with `User not found` if the user has never played a game, even if the user does exist.*
-
-#### Usage Examples
+*Note: all user arguments are case-insensitive, as usernames are case-insensitive.*
 
 ```
-pippi: !stats peppy
-BanchoBot: Stats for peppy:
-BanchoBot: Store: 422,187,979 (#94718)
-BanchoBot: Plays: 7149 (lv65)
-BanchoBot: Accuracy: 87.29%
+!stats <user>
 ```
 
+`!stats` displays the entered user's user statistics (stats). The resulting stats depend on the [game mode](/wiki/Game_Modes) that the entered user is playing (or last played). Although BanchoBot will not display which game mode the stats came from. An example of sending this command is shown below:
```

Comment:
- *user's user statistics* -> *user's game statistics*
- please avoid fragment sentences starting with *Although*: https://jalc.edu/files/uploads/global/english/pdfs/writing_center_tutorials/38_the_proper_use_of_although.pdf
- the word "stats" is mentioned in 3 of 4 sentences, any chance you could cut its usage down?